global/prometheus-infra/templates/_prometheus.yaml.tpl

```diff
@@ -133,6 +133,7 @@
       - '{__name__=~"^node_logind_sessions", class="user", job="jumpserver", type="tty"}'
       - '{__name__=~"^prom_fluentd_.+"}'
       - '{job="netbox", __name__!~"^(up|ALERTS.*|scrape.+)"}'
+      - '{__name__=~"^cc3test_status", service="ironic", type="baremetal_and_regression"}'
```

Comment:
Looks good. But the query doesn't match any time series currently. Is this intended?

Reply:
I see. Looked in two other regions.